### ISSUE TITLE
Notice about age in password screen should be after input in accessibility mode

### ISSUE THREAD

**Issue by Nexcius** [schibsted/account-sdk-android#218]
This has no context in screen readers

**Comment by Nexcius** [comment]
Needs validation that this is the way to go
